### ISSUE TITLE
Get NaN ERR

### ISSUE THREAD

**Issue by YcZhangSing** [andimarafioti/florence2-finetuning#25]
Thank you for such excellent open-source work!
When I was fine-tuning florence2-base for image-text understanding tasks, the length of **the label text did not exceed 260 words**, but the language modeling loss obtained during training **resulted in NaN during backward()**, causing the training to fail.
Do you have any solutions to this problem?
